doc-site/introduction/designElements.md

```diff
@@ -25,5 +41,5 @@ Designsystemet samler og tilgjengeliggjør verktøy, filer, komponenter og desig
 
 Designsystemet samler og tilgjengeliggjør verktøy, filer, komponenter og designretningslinjer som brukes av designere og utviklere for å lage gode og konsekvente brukeropplevelser i NVE og Varsoms digitale tjenester. Designsystemet skal sette NVE og Varsoms profiler i kontekst. Med andre ord vise deg hvorfor, hvor, når og hvordan du bruker og videreutvikler innhold i designsystemet for å skape gevinster og tettere samarbeid på tvers av roller og produkter.
```

Comment:
Typografi-avsnittet ser ikke ut som det er ferdig? Hva med å legge inn en TODO i stedet for en dummy-tekst?